### ISSUE TITLE
Polis Crusher (New Card)

### ISSUE THREAD

**Issue by kelsioboy** [magarena/magarena#1259]
[Polis_Crusher.txt](https://github.com/magarena/magarena/files/1229226/Polis_Crusher.txt)

**Comment by melvinzhang** [comment]
added in 4bef8b1de606d
@kelsioboy do checkout https://github.com/magarena/magarena/wiki/UpcomingCards before starting on new cards to avoid duplicate work.

**Comment by kelsioboy** [comment]
@melvinzhang  Cool, that's what I wanted to know. That way I'll be unprepared to send out cards. Thank you for the link.
